Taskfile.yml

```diff
@@ -232,8 +235,9 @@ tasks:
     vars:
       DOCKER_REPO: '{{ default "localhost:5000" .DOCKER_REPO }}'
       TAG: '{{ default "latest" .VERSION }}'
+      SED: '{{if eq .OS "Windows_NT"}}%{$_ -replace "\.git$",""}{{else}}sed "s/\.git$//g" {{end}}' # Remove ".git" from remote url
       SRC_REPO:
-        sh: git ls-remote --get-url | sed 's/\.git$//g'
+        sh: "{{.SHELL}} 'git ls-remote --get-url | {{.SED}}'"
```

Comment:
Hold on, this one broke for me

Reply:
Removing the enclosing `'` fixes the issue (but perhaps breaks windows?)

Reply:
Yeah.. I'll move the whole thing to a Variable then